src/webview/handleOutboundEvents.js

```diff
@@ -212,12 +213,20 @@ const handleLongPress = (
   }
   const { dispatch, showActionSheetWithOptions, backgroundData, narrow, startEditMessage } = props;
   if (target === 'header') {
-    showHeaderActionSheet(
-      showActionSheetWithOptions,
-      { dispatch, startEditMessage, _ },
```

Comment:
Removing `startEditMessage` here doesn't seem to correspond to anything else in this commit -- there's probably a different commit it belongs in.

Ah indeed, it looks like the function's parameter type only drops this property in the last commit in the branch. So at this commit, I think that means Flow would fail. That means a handy command for spotting this kind of thing is `git rebase --exec 'tools/test flow'` -- that will check out each commit of your branch in turn, and run Flow at it, and if it ever fails it'll return you to a shell there. (From which you can say `git rebase --continue` to proceed to later commits, or `git rebase --abort` to just reset your Git state to where you started.)

Reply:
Oops, fixed :)